### PR TITLE
Implement RemoveUnusedImports rewrite.

### PR DESCRIPTION
**Pull request by olafurpg**
This rewrite does not group/sort/expand rewrites, only remove unused
imports. Supersedes #152

### CODE REVIEW

**Comment by olafurpg** [comment]
Diff from running this rewrite on slick https://github.com/slick/slick/pull/1736

File: bin/new-rewrite.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eux
+rewrite=$1
+
+touch scalafix-core/src/main/scala/scalafix/rewrite/${rewrite}.scala
+touch scalafix-tests/input/src/main/scala/test/${rewrite}.scala
+touch scalafix-tests/output/src/main/scala/test/${rewrite}.scala
```
Comment (gabro) [inline bin/new-rewrite.sh]: ❤️

**Comment by olafurpg** [comment]
@dpennell can you elaborate? This rule is included in both the current stable v0.5.10 release and also the latest v0.6 milestone.

**Comment by dpennell** [comment]
@olafurpg apologies - I've managed to cross-over from scalafmt to scalafix in my search.

**Pull request merged**